docs/templates/runtime.adoc

```diff
@@ -1,7 +1,6 @@
 :source-highlighter: highlight.js
 :highlightjs-languages: rust
 :github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
-= Pallet Name
```

Comment:
I think that `Heading 1` is required as it will be a header for the page. Could you please rename it to something like Runtime Header?